examples/aco_tsp/run_tsp.py

```diff
@@ -0,0 +1,47 @@
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+from aco_tsp.model import AcoTspModel, TSPGraph
+
+
+def main():
+    # tsp_graph = TSPGraph.from_random(num_cities=20, seed=1)
+    tsp_graph = TSPGraph.from_tsp_file("aco_tsp/data/kroA100.tsp")
+    model_params = {
+        "num_agents": tsp_graph.num_cities,
+        "tsp_graph": tsp_graph,
+    }
+    number_of_episodes = 50
+
+    results = defaultdict(list)
+
+    best_path = None
+    best_distance = float("inf")
+
+    model = AcoTspModel(**model_params)
+
+    for e in range(number_of_episodes):
+        # model = AcoTspModel(**model_params)
+        model.step()
+        results["best_distance"].append(model.best_distance)
+        results["best_path"].append(model.best_path)
+        print(
+            f"Episode={e + 1}; Min. distance={model.best_distance}; pheromone_1_8={model.grid.G[17][15]['pheromone']}"
```

Comment:
Prints nicer with a few less decimals
```suggestion
            f"Episode={e + 1}; Min. distance={model.best_distance:.2f}; pheromone_1_8={model.grid.G[17][15]['pheromone']:.4f}"
```